.github/workflows/sidecar-build.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: sidecar-build
+
+on:
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  test:
+    name: test and build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go: ["1.17"]
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
add

    path: /home/runner/go/src/github.com/csi-addons

so that you can skip the next copying step